home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1584,25 +1593,23 @@ components:
       properties:
         first_name:
           type: string
-          example: Petro
+          example: "Petro"
         last_name:
           type: string
-          example: Petrov
+          example: "Petrov"
         email:
           type: string
-          example: home@gmail.com
+          example: "home@gmail.com"
         contacts:
           type: array
           example:
             - type: "email"
-            - id: 1
-            - main: false
-            - email: "readUserEmail@example.com"
+              id: 1
```

Comment:
Can we make examples closer to the backend app we are implementing? I mean if we are using aplication/json mediatype then examples should be of the same format.

Reply:
Perhaps I didn't fully understand the question. But in the end, this example generates a JSON file the same as we get in a real application.

Reply:
I agree that JSON and YAML are interchangeable here. But when we briefly look at the code it would be great to see json here without additional steps to go to the converter and make json from yaml. Please define all defined examples as json.

Reply:
> I agree that JSON and YAML are interchangeable here. But when we briefly look at the code it would be great to see json here without additional steps to go to the converter and make json from yaml. Please define all defined examples as json.

already done